hack/verify-govet.sh

```diff
@@ -5,7 +5,7 @@ set -o pipefail
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.5') ]]; then
```

Comment:
Jenkins doesn't run on 1.5 yet, so if you want this actually enforced, it has to be 1.4 as well